raiden-derive/src/ops/batch_delete.rs

```diff
@@ -92,6 +92,19 @@ pub(crate) fn expand_batch_delete(
         }
     };
 
+    let api_call_token = super::api_call_token!("batch_write_item");
+    let (call_inner_run, inner_run_args) = if cfg!(feature = "tracing") {
+        (
+            quote! { #builder_name::inner_run(&self.table_name, &self.client, input).await? },
+            quote! { table_name: &str, },
+        )
+    } else {
+        (
+            quote! { #builder_name::inner_run(&self.client, input).await? },
+            quote! {},
+        )
+    };
```

Comment:
Create token stream to call DynamoDB API with span.  
if feature `tracing` is not active, token stream will be created without making span.